### DISCUSSION TITLE
Proposal: add a '950' shade for darker colors

### DISCUSSION THREAD

**Discussion by JaimeObregon** [tailwindlabs/tailwindcss#4621] [Ideas]
I really love Tailwind's color palette 🥰 . The only thing I miss is an even darker `950` shade for site backgrounds

As an example, this is the actual Indie Hackers vs its **darkest** possible counterpart using Tailwind's default palette, which feels still too light to me.

Does this make sense? What are your thoughts?

| Actual Indie Hackers | Darkest possible with Tailwind's default palette |
| --- | --- |
| ![1](https://user-images.githubusercontent.com/417226/121637315-7c51d480-ca89-11eb-9786-0ffbf67228d9.jpg) | ![2](https://user-images.githubusercontent.com/417226/121637324-7eb42e80-ca89-11eb-875d-2ebb4f41fe11.jpg) |

**Comment by ttsirkia** [comment]
I assume that the current color palette is optimized to be used in "light mode", I guess that this is more like a "dark mode". I opened another discussion #5215 to highlight the difference between the lightest and darkest color with white and black background.
